x-pack/legacy/plugins/siem/public/pages/case/components/configure_cases/closure_options_radio.test.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { ClosureOptionsRadio } from './closure_options_radio';
+import { useMountAppended } from '../../../../utils/use_mount_appended';
+import { TestProviders } from '../../../../mock';
+
+describe('ClosureOptionsRadio', () => {
+  const mount = useMountAppended();
+
+  test('it renders', () => {
+    const wrapper = shallow(
+      <ClosureOptionsRadio
+        disabled={false}
+        closureTypeSelected="close-by-user"
+        onChangeClosureType={jest.fn()}
+      />
+    );
+
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  test('it shows the correct number of radio buttons', () => {
+    const wrapper = mount(
+      <ClosureOptionsRadio
+        disabled={false}
+        closureTypeSelected="close-by-user"
+        onChangeClosureType={jest.fn()}
+      />,
+      { wrappingComponent: TestProviders }
+    );
+
+    expect(wrapper.find('input[name="closure_options"]')).toHaveLength(2);
+  });
+
+  test('it renders the correct radio buttons', () => {
+    const wrapper = mount(
+      <ClosureOptionsRadio
+        disabled={false}
+        closureTypeSelected="close-by-user"
+        onChangeClosureType={jest.fn()}
+      />,
+      { wrappingComponent: TestProviders }
+    );
+
+    expect(wrapper.find('input[id="close-by-user"]')).toBeDefined();
```

Comment:
We use exists to verify this case in existing code base, e.g.: expect(wrapper.find('input[id="close-by-user"]').exists()).toBeTruthy();